.ci_support/migrations/cuda120.yaml

```diff
@@ -2,6 +2,10 @@ migrator_ts: 1682985063
 __migrator:
   kind:
     version
+  # Vendor CUDA 12 migrator to use GCC 11.
+  # This is needed to workaround a pybind + GCC 12 + nvcc 12 bug
+  # xref: https://github.com/pybind/pybind11/issues/4606
+  use_local: true
```

Comment:
Cherry-picked commit ( https://github.com/conda-forge/pp-sketchlib-feedstock/pull/44/commits/bcf69f5f95559ea493287434a0afb64b6b2f1906 ) from PR ( https://github.com/conda-forge/pp-sketchlib-feedstock/pull/44 ), which was needed as a workaround to upstream issue ( https://github.com/pybind/pybind11/issues/4606 ). More details in issue ( https://github.com/conda-forge/pp-sketchlib-feedstock/issues/43 )